### PR TITLE
PR for BONKLANA

### PR DESCRIPTION
**Pull request by KnucklesButRich**
https://solscan.io/token/88tmABf1s3A6jbLD2yLR1tgbGjz68fz3EDrbeSB1JPPc

https://twitter.com/Bonklana/status/1731772114309415189?s=20

Validation (Please check off boxes):
[+] The metadata provided in the PR matches what is on-chain (Mandatory)
[+] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
[-] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)
Acknowledgement (Please check off boxes)
[+] My change matches the format in the file (no spaces between fields).
[Not Yet, on 11/12/2023 we will be trading on jupiter, but the token is already live. ] My token is already live and trading on Jupiter.
[+] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

File: validated-tokens.csv [hunk 1]
```diff
@@ -1035,4 +1035,4 @@ Who,WHO,A8eYjpNQF6QYEE7DnM4F514MErFzV1pxVT69d7mdSsMk,9,https://bafybeiaahcxwjk6o
 DOBI,DOBI,GzQzkt2B4Jr6whWVBF7XqkzWvoUy1jEd5z9tczzGg1rH,6,https://gateway.irys.xyz/099elCExuWO7iBY2_E-4mZVNLL21Hmj_nxHTGTdUxx0,true
 Lamas Finance,LMF,LMFzmYL6y1FX8HsEmZ6yNKNzercBmtmpg2ZoLwuUboU,9,https://www.lamas.co/resource/lmf_token.png,true
 Rocky,ROCKY,4icEZCrEYNop2ZaMMCkRHaNzkt6xG9BpijMCQV7mpw6Z,9,https://i.ibb.co/X8r9531/rockylogo-transparents.png,true
-
+BONKLANA,BOK,88tmABf1s3A6jbLD2yLR1tgbGjz68fz3EDrbeSB1JPPc,9,https://node1.irys.xyz/oJvtfp7SYicoLG72x16OgkU5YK0SXK-krvF5k8_q0jI,true
```
Comment (Timoon21) [inline validated-tokens.csv:1038]:
```suggestion
BONKLANA,BOK,88tmABf1s3A6jbLD2yLR1tgbGjz68fz3EDrbeSB1JPPc,9,https://gateway.irys.xyz/oJvtfp7SYicoLG72x16OgkU5YK0SXK-krvF5k8_q0jI,true
```

**Review by Timoon21 — APPROVED** [review]

**Comment by KnucklesButRich** [comment]
Hello, can we please request another reviewer, its been a week since invite

**Comment by KnucklesButRich** [comment]
Hello, can we please request another reviewer, its been more than a week since invite

**Comment by KnucklesButRich** [comment]
https://solscan.io/token/88tmABf1s3A6jbLD2yLR1tgbGjz68fz3EDrbeSB1JPPc

https://twitter.com/Bonklana/status/1731772114309415189?s=20

https://www.coingecko.com/en/coins/bonklana

https://coinmarketcap.com/currencies/bonklana/

Validation (Please check off boxes):
[+] The metadata provided in the PR matches what is on-chain (Mandatory)
[+] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
[+] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)
Acknowledgement (Please check off boxes)
[+] My change matches the format in the file (no spaces between fields).
[Not Yet, on 11/12/2023 we will be trading on jupiter, but the token is already live. ] My token is already live and trading on Jupiter.
[+] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

**Comment by KnucklesButRich** [comment]
hello is anybody here?

**Review by 9yoi — APPROVED** [review]

**Comment by allypally19869** [comment]
Please validate this, been 2 months now. @Timoon21

**Pull request merged**